test/c/basic/srem.c

```diff
@@ -0,0 +1,46 @@
+#include "smack.h"
+
+// @expect verified
+
+int main(void) {
+  int x;
+  x = 62951;
+  assert(x % (-37) == 14);
```

Comment:
Clang often does constant propagation and then evaluates expressions where it can. Maybe it does that even with `O0`. Could you check if the generated Boogie actually contains these remainder operations just to make sure they are actually being generated?